R/pkg/vignettes/sparkr-vignettes.Rmd

```diff
@@ -526,6 +530,34 @@ gaussianFitted <- predict(gaussianGLM, carsDF)
 head(select(gaussianFitted, "model", "prediction", "mpg", "wt", "hp"))
 ```
 
+#### Random Forest
+
+`spark.randomForest` fits a [random forest](https://en.wikipedia.org/wiki/Random_forest) classification or regression model on a `SparkDataFrame`.
+Users can call `summary` to get a summary of the fitted model, `predict` to make predictions, and `write.ml`/`read.ml` to save/load fitted models.
+
+In the following example, we use the `longley` dataset to train a random forest and make predictions:
+
+```{r}
```

Comment:
longley has columns with `.` like Armed.Forces
suggest putting `warning = FALSE` like [here](https://github.com/apache/spark/pull/16222/files#diff-56790892076daac0bc43a9b38c0b1470R784) otherwise we will have a warning in the output vignettes